prereqs/az-dcap-client/Makefile

```diff
@@ -1,38 +1,18 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-AZ_DCAP_PKG=az-dcap-client_1.0_amd64.deb
+all: setuprepo
 
-all: getdist
-
-getdist:
-	$(MAKE) -s distclean
-	$(MAKE) get-quote-provider
-
-distclean:
-	- rm -f $(AZ_DCAP_PKG)
-
-build:
+setuprepo:
+	echo "deb [arch=amd64] https://packages.microsoft.com/ubuntu/16.04/prod xenial main" | tee /etc/apt/sources.list.d/msprod.list
+	wget -qO - https://packages.microsoft.com/keys/microsoft.asc | apt-key add -
+	apt-get update
 
 clean:
 
 install:
```

Comment:
Won't this fail if not run as `make && make install`? Since install doesn't depend on setuprepo.

Reply:
(Same for below. Also I might be wrong about make.)

Reply:
Right now the docs say to run:

```bash
sudo make -C prereqs/az-dcap-client
sudo make -C prereqs/az-dcap-client install

sudo make -C prereqs USE_LIBSGX=1
sudo make -C prereqs install USE_LIBSGX=1
```

I agree with you, I don't see a real need to put this in a makefile. We could make install-prereqs scripts that are specific for SGX1+FLC and SGX1

Reply:
I do think that GNU Make is fine for scripts like this (it's a nice scripting language when compared to Bash 😝 but make && make install is totally unnecessary. We should just add `setuprepo` as a dependency of install (and it should have `/etc/apt/sources.list.d/msprod.list` as a dependency so it only runs once).